### ISSUE TITLE
Correção dos ícones de REMOVER SELEÇÃO e DESFAVORITAR

### ISSUE THREAD

**Issue by luizapeixe** [unidoscontraacorrupcao/ej-server#250]
a partir implementação do ícone de Remover Seleção atual é possível perceber um erro na exportação de alguns ícones. 

como ficou:
![captura de tela 2018-09-11 as 19 25 55](https://user-images.githubusercontent.com/6249788/45391223-8b657880-b5f8-11e8-897d-9c42bd4eb8b2.png)

como era pra ser:
![captura de tela 2018-09-11 as 19 25 33](https://user-images.githubusercontent.com/6249788/45391201-7983d580-b5f8-11e8-86d3-5115f69ae2ec.png)

Então segue abaixo a relação de ícones corrigidos e respectivos links:

- [remover seleção](https://drive.google.com/file/d/1GcWJhUdWJae9dCOzZb-o8T0lHBKSlgWc/view?usp=sharing) 
- [remover seleção 2](https://drive.google.com/file/d/1stO9hJwIqZajfuJJEnFMm6bTRYubwdYq/view?usp=sharing) 
- [desfavoritar](https://drive.google.com/file/d/1OHCTN7EN-0QFJC3_90udHWPe2HSkBAq9/view?usp=sharing)

lembrando que na próxima fase provavelmente vamos alterar alguns ícones a pedido do cliente, então já corrigi também os ícones de remover seleção no novo estilo:

- [remover seleção V2](https://drive.google.com/file/d/1VVTzKT9sJcU6DJPQwLnclmZBiVE-O_ld/view?usp=sharing)
- [remover seleção 2 V2](https://drive.google.com/file/d/15WL9iZmu5Kfv6I5YqGQjsCDyR24BK3Vf/view?usp=sharing)

cc @davidCarlos @danielhmarinho

**Comment by ricardopoppi** [comment]
@luizapeixe acho que a essa altura não vamos mais implementar essa v2 dos ícones, apenas corrigir os que lá estão. digo isso pra gente focar em melhorias de ux mesmo. confere @parrahenri?

**Comment by ricardopoppi** [comment]
@danielhmarinho sobre essa issue, vale apenas para o botão de remover seleção, pois a funcionalidade favoritar está em hold, haverá mudança de nome, então o botão não servirá mais

**Comment by parrahenri** [comment]
Sim, uma das questões é que na interna há o contador de "favoritos". temos de pensar a repercussão pra essa tela tb. Embora o ideal é o mesmo nome no ícone e nesses lugares, talvez teremos de manter "favoritos" na interna e "santinho" em outros lugares?

**Comment by ricardopoppi** [comment]
Podemos trocar esse contador pra SANTINHOS. Mantém a coerência.
